### DISCUSSION TITLE
Include the first paragraph of the article in the notification.

### DISCUSSION THREAD

**Discussion by KGIII** [hizzle-co/noptin#582] [Ideas]
Rather than messing about with excerpts and stuff like that, a great feature would be if you could include the first paragraph by default. I don't think I'd be alone in appreciating this sort of feature. I've been meaning to mention this for like a year now, but I keep forgetting.

_I don't know if it's possible._ It seems like it might be.

Maybe make it a buy-it-once feature where it's like $10 or $20 and there's no subscription? (You've added a couple of my requested features so far, but I legit have no use for any of your paid offerings. This would give me a reason to send you a few bucks.)

Again, I doubt I'd be alone in wanting this. Maybe make it [[first_paragraph]] and let folks access it that way? It'd be entirely up to y'all as to how you implement it.

**Comment by picocodes** [comment]
I've added this for the next update.

You'll be able to use it like:-

```
[[post.content paragraphs=1]]
```

Replace 1 with the number of paragraphs you want to display.
